pkg/istio/rbac/v2/provider_test.go

```diff
@@ -55,7 +56,19 @@ func TestConvertAthenzModelIntoIstioRbac(t *testing.T) {
 
 	convertedCR := p.ConvertAthenzModelIntoIstioRbac(domainRBAC, onboardedService.Name, labels["app"])
 	expectedCR := getExpectedCR()
-	assert.Equal(t, expectedCR, convertedCR, "converted authz policy should be equal")
+	// when there are mulitple assertions matched with svc,
+	// base on which assertion will be processed from the athenzModel mapping, order of the authz rule generated
+	// may be different each time. Using a workaround here to sort the configSpec in alphabetical order of first
+	// method in each rule.
+	configSpec := (convertedCR[0].Spec).(*v1beta1.AuthorizationPolicy)
+	rules := configSpec.Rules
+	sort.Slice(rules, func(i, j int) bool {
```

Comment:
I believe sort.Slice directly modifies the underlying array so we could just call this function using configSpec.Rules and remove the temporary variables.